app/controllers/detailedvaluationrequest/DvrController.scala

```diff
@@ -81,7 +81,7 @@ class DvrController @Inject()(
           optDocuments <- dvrCaseManagement.getDvrDocuments(link.uarn, valuationId, link.submissionId)
           backUrl      <- calculateBackLink(propertyLinkSubmissionId, owner)
           startCheckUrl = config.businessRatesCheckUrl(
-            s"property-link/${link.authorisationId}/assessment/$valuationId?propertyLinkSubmissionId=$propertyLinkSubmissionId&uarn=$uarn&dvrCheck=true")
+            s"property-link/${link.authorisationId}/assessment/$valuationId?propertyLinkSubmissionId=$propertyLinkSubmissionId&uarn=$uarn&dvrCheck=true&isOwner=$owner")
```

Comment:
Just me to understand, is it fixing existing bug or added new query parameter "isOwner=$owner"
If new  query parameter "isOwner=$owner", I hope already business rates check can handle this query paramter

Reply:
good spot Reddy. I think I might not have had the latest master. I will update from master

Reply:
good spot Reddy, that should not have been committed. That code was from when I was trying to fix another bug. Im updating the PR